apps/web-client/src/lib/db/cr-sqlite/order-reconciliation.ts

```diff
@@ -204,6 +204,39 @@ export async function finalizeReconciliationOrder(db: DB, id: number) {
 	});
 }
 
+export async function getNonReconcilingSupplierOrders(db: DB) {
```

Comment:
- This is difficult to read - how about `getUnreconciledSupplierOrders`? 
- Add tests
- Add JSDoc comment